aws_lambda_powertools/utilities/parser/models/s3.py

```diff
@@ -77,12 +77,12 @@ class S3EventNotificationEventBridgeDetailModel(BaseModel):
     request_id: str = Field(None, alias="request-id")
     requester: str
     source_ip_address: str = Field(None, alias="source-ip-address")
-    reason: Optional[str]
-    deletion_type: Optional[str] = Field(None, alias="deletion-type")
-    restore_expiry_time: Optional[str] = Field(None, alias="restore-expiry-time")
-    source_storage_class: Optional[str] = Field(None, alias="source-storage-class")
-    destination_storage_class: Optional[str] = Field(None, alias="destination-storage-class")
-    destination_access_tier: Optional[str] = Field(None, alias="destination-access-tier")
+    reason: Optional[str] = None
+    deletion_type: Union[str, None] = Field(None, alias="deletion-type")
+    restore_expiry_time: Union[str, None] = Field(None, alias="restore-expiry-time")
+    source_storage_class: Union[str, None] = Field(None, alias="source-storage-class")
+    destination_storage_class: Union[str, None] = Field(None, alias="destination-storage-class")
+    destination_access_tier: Union[str, None] = Field(None, alias="destination-access-tier")
```

Comment:
```suggestion
    reason: Optional[str] = None
    deletion_type: Optional[str] = Field(None, alias="deletion-type")
    restore_expiry_time: Optional[str] = Field(None, alias="restore-expiry-time")
    source_storage_class: Optional[str] = Field(None, alias="source-storage-class")
    destination_storage_class: Optional[str] = Field(None, alias="destination-storage-class")
    destination_access_tier: Optional[str] = Field(None, alias="destination-access-tier")
```

Reply:
I'm curious about this suggestion.
Both options are widely used and accepted in Python to represent an `Optional` field and I wonder if you suggest this because `Optional` is more explicit than `Union`.

Reply:
Just be consistent. The other code uses `Optional`.

Reply:
@Wurstnase I agree with that and reverted all those changes.